src/main/java/wolox/training/security/PasswordEncoder.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.security;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Configuration
+public class PasswordEncoder {
+
+    @Bean
+    public org.springframework.security.crypto.password.PasswordEncoder encoder() {
+        return new BCryptPasswordEncoder();
+    }
+    public String encode(String string){
```

Comment:
Is this method really used? when you create a java Bean and it's injected using Autowired then you get the object returned in the @Bean method ( BCryptPasswordEncoder for this case ) so any method will be defined by the object and not by de config class ( unless you override them )

Reply:
I used in order to mock it in tests,but I could find another way to mock it

Reply:
Oh, now I get it ! And there was no need for the method haha! Thanks